### PR TITLE
Allow to configure how `Entity` updates are delivered to a `Stand`

### PR DESCRIPTION
**Pull request by armiol**
Summary of changes:
* Existing `...Delivery` hierarchy refactored; new base `Delivery` class introduced.
* `StandUpdateDelivery` is created to regulate the propagation of the `Entity` state updates through the `StandFunnel` to the `Stand`.
* `BoundedContext.Builder` no longer allows to set a `standFunnelExecutor`; it allows to set a custom `StandUpdateDelivery` instance instead.
* Spine version increased to `0.7.6-SNAPSHOT`.

UPD:

* `SubscriberRegistry` and `DispatcherRegistry` (the one for `EventDispatcher`s) are updated to return an immutable set of the registered items per key.

### CODE REVIEW

**Comment by armiol** [comment]
@alexander-yevsyukov PTAL.

File: server/src/main/java/org/spine3/server/stand/StandUpdateDelivery.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+         */
+        private static final StandUpdateDelivery DIRECT_DELIVERY = new StandUpdateDelivery() {
+
+            @Override
+            protected boolean shouldPostponeDelivery(Entity deliverable, Stand consumer) {
+                return false;
+            }
+        };
+    }
+
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/stand/StandUpdateDelivery.java]: Please remove excessive empty line.

File: server/src/main/java/org/spine3/server/delivery/Delivery.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    private static <T> Predicate<T> matchClass(final Class<? extends T> classToMatch) {
+        return new Predicate<T>() {
+            @Override
+            public boolean apply(@Nullable T input) {
+                checkNotNull(input);
+                return classToMatch.equals(input.getClass());
+            }
+        };
+    }
+
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/delivery/Delivery.java]: Please remove the empty line here.

File: server/src/main/java/org/spine3/server/stand/StandUpdateDelivery.java (hunk 1 above)
Comment (armiol) [inline server/src/main/java/org/spine3/server/stand/StandUpdateDelivery.java]: Done.

File: server/src/main/java/org/spine3/server/delivery/Delivery.java (hunk 2 above)
Comment (armiol) [inline server/src/main/java/org/spine3/server/delivery/Delivery.java]: Done.

**Pull request merged**